packages/core/admin/admin/src/components/UnstableGuidedTour/Tours.tsx

```diff
@@ -149,7 +149,7 @@ const tours = {
             id="tours.contentManager.FinalStep.content"
             defaultMessage="Now that you’ve created and published content, time to create API tokens and set up permissions."
           />
-          <Step.Actions showStepCount={false} to="/settings" />
+          <Step.Actions showStepCount={false} to="/settings/api-tokens" />
```

Comment:
Are you sure it is gonna to open the api tokens instead of the settings page?

Reply:
Don't we want it to open the api-tokens page? otherwise the use is on the settings page and not sure what to do next

Reply:
yeah but the button in the left menu is pointing the settings page not the api tokens

Reply:
ok